enacts/wat_bal/agronomy.py

```diff
@@ -214,3 +214,109 @@ def soil_plant_water_balance(
             - (planted_since - np.timedelta64(1, "D"))
         )
     return sm, drainage, et_crop, planting_date
+
+
+def weekly_api_runoff(
+    daily_rain,
+    no_runoff=12.5,
+    api_thresh=xr.DataArray([6.3, 19, 31.7, 44.4, 57.1, 69.8], dims=["api_cat"]),
```

Comment:
Isn't it strange that the default `api_thresh[0]` is smaller than the default `no_runoff`? Isn't that giving us a non-zero polynomial to calculate the runoff in cases where there's no runoff?

Should we avoid such inconsistencies and simplify the cold by folding no_runoff into api_poly, prepending an entry with api_thresh = no_runoff and api_poly = (0, 0, 0)?

Reply:
You actually did fold no_runoff into api_thresh in the `select` version.